From: =?utf-8?b?IlNaIExpbiAo5p6X5LiK5pm6KSI=?= <szlin@debian.org>
Date: Tue, 13 Nov 2018 16:16:23 +0800
Subject: Fix macro warning/ error by adding escaped symbol

---
 man/tpm2-abrmd.8.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/man/tpm2-abrmd.8.in b/man/tpm2-abrmd.8.in
index 5e96053..44a44b8 100644
--- a/man/tpm2-abrmd.8.in
+++ b/man/tpm2-abrmd.8.in
@@ -92,15 +92,15 @@ This is equivalent to:
 .B tpm2-abrmd --tcti="libtss2-tcti-device.so.0:/dev/tpm0"
 .TP
 Have daemon use Microsoft/IBM TPM2 Simulator tcti library
-'libtss2-tcti-mssim.so.0'.
+\'libtss2-tcti-mssim.so.0\'.
 This connects to a TPM2 simulator via a TCP mssim.
 .br
 .B tpm2-abrmd --tcti="mssim"
 .br
 .B tpm2-abrmd --tcti="libtss2-tcti-mssim.so.0"
 .TP
-Have daemon use tcti library 'libtss2-tcti-mssim.so.0' and config string
-'host=127.0.0.1,port=5555':
+Have daemon use tcti library \'libtss2-tcti-mssim.so.0\' and config string
+\'host=127.0.0.1,port=5555\':
 .B tpm2-abrmd --tcti=mssim:host=127.0.0.1,port=5555"
 .br
 .B tpm2-abrmd --tcti="libtss2-tcti-mssim.so.0:host=127.0.0.1,port=5555"
